client.go

```diff
@@ -272,3 +295,70 @@ func (cl *Client) Namespaces(ctx context.Context) ([]Namespace, error) {
 
 	return ns, nil
 }
+
+// Users get list of users by id.
+func (cl *Client) Users(ctx context.Context, ids ...int) (map[int]User, error) {
+	ususerids := []string{}
+
+	for _, id := range ids {
+		ususerids = append(ususerids, strconv.Itoa(id))
```

Comment:
it would've been nice to check if strconv.Itoa returns an error:
newid, err := strconv.Itoa(id)

Reply:
`strconv.Itoa` does not return error (`Atoi` does but we don't use it in this piece of code). 
https://pkg.go.dev/strconv#Itoa

Reply:
ok